Source/WebCore/rendering/svg/RenderSVGResourceFilter.cpp

```diff
@@ -87,6 +87,11 @@ bool RenderSVGResourceFilter::applyResource(RenderElement& renderer, const Rende
 
     if (m_rendererFilterDataMap.contains(&renderer)) {
         FilterData* filterData = m_rendererFilterDataMap.get(&renderer);
+        if (!filterData->sourceImage) {
+            auto styles = filterData->filter->createStyles();
+            context->addFilterEffectStyles(styles);
```

Comment:
This will change too. But why is the code not consulting the mode?

Reply:
Done.